src/components/Checkout/DonationItem.vue

```diff
@@ -32,6 +32,14 @@
 					v-kv-track-event="['basket', 'Donation Info Lightbox', 'Open Lightbox']">
 					How Kiva uses donations
 				</a>
+				<donation-nudge-lightbox
```

Comment:
I don't think this should be rendered unless you have an assignment returned on the donationNudgeLightboxExpVersion variable. If that is null the exp is not enabled so we shouldn't inject the contnet.

Reply:
Probably not an issue seeing as this is the new nudge and replaces the simple input field when loans are in the basket. No action needed.